extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -12,6 +12,7 @@ import RX = require('reactxp');
 
 export interface VirtualListCellInfo {
     key: string;
+    onScreenChanged?: (isOnScreen: boolean) => void;
```

Comment:
The terminology here isn't ideal. "Screen" usually implies the full physical display surface, whereas this is referring to the view port of the list.